eng/Versions.props

```diff
@@ -1,6 +1,11 @@
 <Project>
   <!-- Import references updated by Dependabot. This file is for package references updated manually or by Maestro. -->
   <Import Project="dependabot/Versions.props" />
+  <Import Project="dependabot/nuget.org/Versions.props" />
+  <Import Project="dependabot/net6.0/Versions.props" />
+  <Import Project="dependabot/net7.0/Versions.props" />
+  <Import Project="dependabot/net8.0/Versions.props" />
+  <Import Project="dependabot/netcoreapp3.1/Versions.props" />
```

Comment:
Note: This isn't technically correct.  Once https://github.com/dotnet/dotnet-monitor/pull/6071 is backported to other branches this problem will go away.